challenge-05/challenge-05.js

```diff
@@ -2,17 +2,19 @@
 Crie uma variável qualquer, que receba um array com alguns valores aleatórios
 - ao menos 5 - (fica por sua conta os valores do array).
 */
-// ?
+var arr = [2, false, 'Tannus', 2.4, undefined];
 
 /*
 Crie uma função que receba um array como parâmetro, e retorne esse array.
 */
-// ?
+function paramArr(arr) {
+  return arr;
+};
```

Comment:
-1
Após funções literais, estruturas condicionais e de repetição (for, while, etc), não usa-se ponto-e-vírgula no final. Somente quando você for atribuir uma função à uma variável:

```js
function myFunction() {} // não usa-se ponto-e-vírgula, pois é uma função literal
var myFunction = function() {}; // aqui usa-se, pois estamos atribuindo à uma variável
```

:)